editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -1299,10 +1303,15 @@ EditorAssetLibrary::EditorAssetLibrary(bool p_templates_only) {
 	filter = memnew(LineEdit);
 	search_hb->add_child(filter);
```

Comment:
Is the hbox still needed actually?

Reply:
Yes, it's still used for the **Import…** and **Plugins…** buttons while in the editor:

![image](https://user-images.githubusercontent.com/180032/94556716-d5f8d280-025d-11eb-830d-d313c36ee8bf.png)